.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
I'd prefer style of having one space after the dash. (here and in all places below)